### ISSUE TITLE
Radiation belt event strange behaviour

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#34289]
(WEB REPORT BY: inorien REMOTE: 206.221.180.138:7777)
# Revision
66e3f972de6fbe5ceaffb551813b6475dd665db6

# Description

Radiation belt event started but only some rooms became lit up (flashing green). Going near these dark rooms would cause them to light up and begin flashing.

After the event completed, these rooms are still flashing green. Going near them clears this.

# Steps to Reproduce

Radiation belt event
Just walk around before and after (or during I guess)

# What you Expected

Green flashing lights to behave normally

# What Actually Happened

No such thing

**Comment by Inorien** [comment]
Note theres been no merges since I last saw this working, so likely something else causing it
I guess I can post details after this round with 1 person in it finishes

**Comment by Eneocho** [comment]
Had this happen to me on a round today. The room didn't flash so I thought it was safe. I almost died to rads.
The nearby rooms kept flashing even after the rads passed, and only updated when I exited maint and got to that hallway.

**Comment by adacovsk** [comment]
Was there no power in that room?

**Comment by Eneocho** [comment]
It was post engineering shart exploded I think, but I'm not sure now.

**Comment by Inorien** [comment]
confirmed this happens independently of room power state

it seems that the affected rooms are the same for everyone, ie. if medbay isn't flashing when it should be, everyone sees the same

**Comment by nervere** [comment]
This happens all the time. Able to replicate on local easily. The area's `icon_state` and `radalert` is being set properly, but the area's appearance doesn't update properly regardless. Calling `updateicon()` again doesn't fix anything either. Entering the area seems to fix the incorrect appearance.

@Exxion Any ideas?

**Comment by Inorien** [comment]
peace of shit byond fucking trash lummox i hate you

**Comment by Exxion** [comment]
This is probably a BYOND bug fixed in 515.1605.

**Comment by pedr69** [comment]
when the radstorm ends none of the room glows get reset until a player walks into them

**Comment by Evige** [comment]
I have been facing the issue which was described by Eneocho "The nearby rooms kept flashing even after the rads passed, and only updated when I exited maint and got to that hallway.". We had a round with multiple radiation belt random events. I noticed that the lights kept flickering after the first event and the same issue then happened after all the radiation belt events during the same game. 

For clarification: I haven't faced the original issue of this ticket even once, so I think that is fixed, but still that other flickering keeps happening after the events.

BYOND version: 515.1635.

**Comment by Eneocho** [comment]
yeah the flashing lights being fucked until someone enters the room thing keeps happening
